SEED-platform/seed
# 3269 fix/update property list hamburger menu

#### Any background context you want to provide?
see ticket

#### What's this PR do?
Updates the (beta) inventory list hamburger export actions to work identical to the page's export action.

#### How should this be manually tested?
- Select 0 properties and click the "Export All/Selected" button from the hamburger, make sure the resulting export has all properties.
- Select 1 or more  properties and click the "Export All/Selected" button from the hamburger, make sure the resulting export has selected properties.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/3269

#### Screenshots (if appropriate)
New menu with consolidated export options:
<img width="1225" alt="Screen Shot 2022-07-06 at 4 46 59 PM" src="https://user-images.githubusercontent.com/564231/177655910-44c8a662-70bc-4193-a6af-156de270c64e.png">

Clicking opens the export modal:
<img width="1214" alt="Screen Shot 2022-07-06 at 4 47 06 PM" src="https://user-images.githubusercontent.com/564231/177655913-69dc0ddd-8fca-437c-bf9c-9458d024bfeb.png">
